code-challenges.test.js

```diff
@@ -53,4 +130,29 @@ const numbersToAdd3 = []
 // Expected output: []
 
 
+describe ("sumNums", () => {
+  const numbersToAdd1 = [2, 4, 45, 9]
+  const numbersToAdd2 = [0, 7, -8, 12]
+  it ("array of the accumulating sum, empty array should return an empty array", () => {
+  expect (sumNums(numbersToAdd1)).toEqual([2, 6, 51, 60])
+  expect (sumNums(numbersToAdd2)).toEqual([0, 7, -1, 11])
+  expect (sumNums(numbersToAdd3)).toEqual([])
+  })
+
+})
+
+
+
+
 // b) Create the function that makes the test pass.
+
+//pusedo code
+//create a function called SumNums
+//create a parameter called array
+//use .map to iterate through the arry
+// apply a condition to return the new array with numbers added from the sum of the old array to the new array
+
+const sumNums = (array) => {    let newArray = []  
+    return array.map((value, index) => {        if(index > 0){            return newArray = value + newArray        }else {            return newArray = value        }    })}
```

Comment:
Your logic looks great, just make sure to watch how this is formatted and indentation.

Reply:
got it!!